HousingFinanceInterimApi/Startup.cs

```diff
@@ -173,7 +170,7 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env, I
                 app.UseHsts();
             }
 
-            app.UseCors(options => options.WithOrigins("http://localhost:44335", "https://dmg8fqy2zxv7c.cloudfront.net")
+            app.UseCors(options => options.WithOrigins("http://localhost:3000", "https://dmg8fqy2zxv7c.cloudfront.net", "https://eoy-report-development.hackney.gov.uk")
```

Comment:
We should have this as an environment variable.  This would more than likely change depending on the environment.

Reply:
Done